web/src/styles/Modal.css

```diff
@@ -128,7 +128,6 @@ input {
     margin: 0.3em 0;
     background-color: white;
     color: #000000;
-    border: none;
```

Comment:
@Victoria-Williamson I took out this border property to allow the date-time field to have a red border drawn around it for form validation.